src/Ykush.js

```diff
@@ -28,12 +28,12 @@ class Ykush {
     }
 
     async powerAllOn() {
-        const args = [...this._prefix, '-s', this._serialNumber, '-u', 'a'];
+        const args = [...this._prefix, '-s', this._serialNumber.id, '-u', 'a'];
```

Comment:
This should be fixed in otherway and separate PR. `list()` function should return list of serialnumber instead of list of objects with id field.

Reply:
Agreed